<?php

return array(    
    'taxa2dist-function-title'  =>  'Calculation of taxonomic distances',
    'taxa2dist-about'           =>  'This function calculates the averaged taxonomic distances among species or individuals in the community from a classification table with variable step lengths. The output of taxa2dist is needed as an input to taxondive function.', 
    'taxa2dist-box'             =>  '', 
    'taxa2dist-varstep'         =>  'If TRUE, varies step lengths between successive levels relative to proportional loss of the number of distinct classes.', 
    'taxa2dist-check_taxa2dist' =>  'If TRUE, removes all redundant levels which are different for all rows or constant for all rows and regard each row as a different basal taxon (species). If FALSE all levels are retained and basal taxa (species) also must be coded as variables (columns).',
    'taxondive-function-title'  =>  'Indices of Taxonomic Diversity and Distinctness',
    'taxondive-about'           =>  'This function computes the indices of taxonomic diversity and taxonomic distinctness. Taxonomic diversity can be thought of as the average taxonomic ‘distance’ between any two organisms, chosen at random from the sample: this distance can be visualized simply as the length of the path connecting these two organisms, traced through (say) a Linnean or phylogenetic classification of the full set of species involved (Clarke & Warwick 1998, 1999, 2001; Pienkowski et al., 1998). Taxonomic distinctness is the average path length between any two randomly chosen individuals, conditional on them being from different species. This is equivalent to dividing taxonomic diversity, by the value it would take were there to be no taxonomic hierarchy (all species belonging to the same genus). It can therefore be seen as a measure of pure taxonomic relatedness (Clarke & Warwick 1998, 1999, 2001; Pienkowski et al., 1998).
	References:
    Clarke, K.R & Warwick, R.M. (1998) A taxonomic distinctness index and its statistical properties. Journal of Applied Ecology 35, 523–531.
    Clarke, K.R. & Warwick, R.M. (1999) The taxonomic distinctness measure of biodiversity: weighting of step lengths between hierarchical levels. Marine Ecology Progress Series 184, 21–29.
    Clarke, K.R. & Warwick, R.M. (2001) A further biodiversity index applicable to species lists: variation in taxonomic distinctness. Marine Ecology Progress Series 216, 265–278.
    Pienkowski, M.W., Watkinson, A.R., Kerby, G., Clarke, K.R., & Warwick, R.M. (1998) A taxonomic distinctness index and its statistical properties. Journal of Applied Ecology, 35(4), 523-531.',
    'taxondive-box'             =>  '',
    'taxondive-box2'            =>  '',
    'taxondive-box3'            =>  '',
    'taxondive-transf_method_select'    =>  'Select a transformation method for the community data matrix. Available options are: none (for no transformation), max (divide by margin maximum (default MARGIN = 2)), freq (divide by margin maximum and multiply by the number of non-zero items, so that the average of non-zero entries is one (default MARGIN = 2)), normalize (make margin sum of squares equal to one (default MARGIN = 1)), range (standardize values into range 0 ... 1 (default MARGIN = 2). If all values are constant, they will be transformed to 0), standardize (scale x to zero mean and unit variance (default MARGIN = 2)), pa (scale x to presence/absence scale (0/1)), chi.square (divide by row sums and square root of column sums, and adjust for square root of matrix total. When used with the Euclidean distance, the distances should be similar to the Chi-square distance used in correspondence analysis. However, the results from cmdscale would still differ, since CA is a weighted ordination method (default MARGIN = 1)), horn, hellinger (square root of method = "total") and log (logarithmic transformation: log_b (x) + 1 for x > 0, where b is the base of the logarithm; zeros are left as zeros. Higher bases give less weight to quantities and more to presences, and logbase = Inf gives the presence/absence scaling. Please note this is not log(x+1).)',
    'taxondive-transpose'       =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'taxondive-column_select'   =>  'Select which column of the factor file you provided will be used as the colour coding factor of the plot.',
    'taxondive-match_force'     =>  'Force matching of column names in community data matrix and labels in the distance matrix. If FALSE, matching only happens when dimensions differ, and in that case the species must be in identical order in both.',
    'taxondive-deltalamda'	=>	'Select if you want to plot the index of taxonomic distinctness (Delta^+) or the index of variation in taxonomic distinctness (Lambda^+).',
    'vegdist-function-title'    =>  'Computation of dissimilarity indices',
    'vegdist-about'             =>  'This function computes dissimilarity indices that are useful for community ecologists. Gower, Bray-Curtis, Jaccard and Kulczynski indices are good in detecting underlying ecological gradients (Faith et al. 1987). Morisita, Horn-Morisita, Binomial, Cao and Chao indices should be able to handle different sample sizes (Wolda 1981, Krebs 1999, Anderson & Millar 2004), and Mountford (1962) and Raup-Crick indices for presence--absence data should be able to handle unknown (and variable) sample sizes. The output of vegdist is needed as an input to hclust and mantel functions. 
	References:
	Anderson, M.J. & Millar, R.B. (2004) Spatial variation and effects of habitat on temperate reef fish assemblages in northeastern New Zealand. Journal of Experimental Marine Biology and Ecology 305, 191-221.
	Faith, D.P., Minchin, P.R. & Belbin, L. (1987) Compositional dissimilarity as a robust measure of ecological distance. Vegetatio 69, 57-68.
	Krebs, C.J. (1999) Ecological Methodology. Addison Wesley Longman.
	Mountford, M.D. (1962) An index of similarity and its application to classification problems. In: P.W.Murphy (ed.), Progress in Soil Zoology, 43-50. Butterworths.
    Wolda, H. (1981) Similarity indices, sample size and diversity. Oecologia 50, 296-302.',
    'vegdist-box'               =>  '',
    'vegdist-transf_method_select'  =>  'Select a transformation method for the community data matrix. For an explanation of all the possibilities, please visit the taxondive fuction.',
    'vegdist-transpose'         =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'vegdist-method_select'     =>  'Select the dissimilarity index you want to compute.',
    'vegdist-binary_select'     =>  'Perform presence/absence standardization before analysis.',
    'vegdist-diag_select'       =>  'Compute diagonals.',
    'vegdist-upper_select'      =>  'Return only the upper diagonal.',
    'vegdist-na_select'         =>  'Pairwise deletion of missing observations when computing dissimilarities.',
    'hclust-function-title'     =>  'Hierarchical cluster analysis on a set of dissimilarities and methods for analyzing it.',
    'hclust-about'              =>  'This function performs a hierarchical cluster analysis using a set of dissimilarities for the n objects being clustered. Initially, each object is assigned to its own cluster and then the algorithm proceeds iteratively, at each stage joining the two most similar clusters, continuing until there is just a single cluster.',
    'hclust-box'                =>  '',
    'hclust-column_select'   =>  'Select which column of the factor file you provided will be used as the colour coding factor of the plot.',
	'hclust-method_select'      =>  'Select the agglomeration method to be used. This should be (an unambiguous abbreviation of) one of "ward.D", "ward.D2", "single", "complete", "average" (= UPGMA), "mcquitty" (= WPGMA), "median" (= WPGMC) or "centroid" (= UPGMC).',
    'bict-function-title'       =>  'Biological Index Calculation Tool',
    'bict-about'                =>  'This function calculates biological indices (Number of species (S), Number of individuals (N), Benthic Quality Index (BQI), Benthic Quality Index for the family level (BQIfam)) frequently used in ecological, conservation and environmental health assessment studies.',
    'bict-box'                  =>  '',
    'bict-box2'                 =>  '',
    'bict-transf_method_select' =>  'Select a transformation method for the community data matrix. For an explanation of all the possibilities, please visit the taxondive fuction.',   
	'bict-transpose'         =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'bict-species_family_select'=>  'Select if you want the indices calculated at the species or family level.',
    'pca-function-title'        =>  'Principal Components Analysis',
    'pca-about'                 =>  'This function performs a principal components analysis on the given numeric data matrix.',
    'pca-box'                   =>  '',
    'pca-box2'                  =>  '',
    'pca-transf_method_select'  =>  'Select a transformation method for the community data matrix. For an explanation of all the possibilities, please visit the taxondive fuction.',
    'pca-transpose'             =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'pca-column_select'         =>  'Select which column of the factor file you provided will be used as the colour coding factor of the plot.',
    'cca-function-title'        =>  'Constrained Correspondence Analysis',
    'cca-about'                 =>  'This function performs Constrained Correspondence Analysis (a.k.a. canonical correspondence analysis) (ter Braak 1986), which is one of the most popular ordination methods in community ecology.
	References:
	Ter Braak, C.J.F. (1986) Canonical Correspondence Analysis: a new eigenvector technique for multivariate direct gradient analysis. Ecology 67, 1167-1179.',
    'cca-box'                   =>  '',
    'cca-box2'                  =>  '',
    'cca-transf_method_select'  =>  'Select a transformation method for the community data matrix. For an explanation of all the possibilities, please visit the taxondive fuction.',
    'cca-transpose'             =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'cca-Factor_select1'        =>  'Select which column of the factor file you provided will be used be used as the first contraint of the analysis.',
    'cca-Factor_select2'        =>  'Select which column of the factor file you provided will be used be used as the second contraint of the analysis.',
    'cca-Factor_select3'        =>  'Select which column of the factor file you provided will be used be used as the third contraint of the analysis.',
    'anosim-function-title'     =>  'Analysis of Similarities',
    'anosim-about'              =>  'This function provides a way to test statistically whether there is a significant difference between two or more groups of sampling units.',
    'anosim-box'                =>  '',
    'anosim-box2'               =>  '',
    'anosim-transf_method_select'   =>  'Select a transformation method for the community data matrix. For an explanation of all the possibilities, please visit the taxondive fuction.',
	'anosim-transpose'             =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'anosim-column_select'      =>  'Select which column of the factor file you provided will be used as the grouping variable of the sampling units.',
    'anosim-permutations'       =>  'Number of permutation to assess the significance of the ANOSIM statistic.',
    'anosim-method_select'      =>  'Select the distance metric that will measure the dissimilarity between each pair of observations. For an explanation of all the possibilities, please visit the vegdist fuction.',
    'anova-function-title'      =>  'Analysis of Variance',
    'anova-about'               =>  'This function is used to determine whether there are any significant differences between the means of two or more independent (unrelated) groups (although you tend to only see it used when there are a minimum of three, rather than two groups). Explanatory variables are categorical (called factors).',
    'anova-box'                 =>  '',
    'anova-Factor_select1'      =>  '',
    'anova-Factor_select2'      =>  '',
    'anova-Factor_select3'      =>  '',
    'permanova-function-title'  =>  'Permutational Multivariate Analysis of Variance Using Distance Matrices',
    'permanova-about'           =>  'This function (adonis) is used for analysis and partitioning of sums of squares using semimetric and metric distance matrices. It is referred to as the “permutational manova” (formerly “nonparametric manova”) (Anderson 2001). Further, as its inputs are linear predictors, and a response matrix of an arbitrary number of columns (2 to millions), it is a robust alternative to both parametric MANOVA and to ordination methods for describing how variation is attributed to different experimental treatments or uncontrolled covariates. It is also analogous to redundancy analysis (Legendre & Anderson 1999).
	References:
    Anderson, M.J. (2001) A new method for non-parametric multivariate analysis of variance. Austral Ecology 26, 32–46.
	Legendre, P. & Anderson, M.J. (1999) Distance-based redundancy analysis: Testing multispecies responses in multifactorial ecological experiments. Ecological Monographs 69, 1–24.',
    'permanova-box'             =>  '',
    'permanova-box2'            =>  '',
    'permanova-transf_method_select'    =>  'Select a transformation method for the community data matrix. For an explanation of all the possibilities, please visit the taxondive fuction.',
	'permanova-transpose'             =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'permanova-column_select'   =>  'Select which column of the factor file you provided will be used as the grouping variable of the sampling units.',
    'permanova-column_select2'  =>  'Select which column of the factor file you provided will be used as the second grouping variable of the sampling units, in case you choose the multiple parameter option.',
    'permanova-permutations'    =>  'Number of replicate permutations used for the hypothesis tests (F tests).',
    'permanova-method_select'   =>  'Select the method used to calculate pairwise distances between each pair of observations. For an explanation of all the possibilities, please visit the vegdist fuction.',
    'mantel-function-title'     =>  'Mantel Tests for Dissimilarity Matrices',
    'mantel-about'              =>  'This function mantel finds the Mantel statistic as a matrix correlation between two dissimilarity matrices. The significance of the statistic is evaluated by permuting rows and columns of the first dissimilarity matrix. Mantel statistic is simply a correlation between entries of two dissimilarity matrices (some use cross products, but these are linearly related). However, the significance cannot be directly assessed. It is equivalent of the RELATE routine in PRIMER software package (Clarke & Gorley, 2006).
	References:
	Clarke, K. & Gorley, R. (2006) PRIMER v6: User Manual/Tutorial PRIMER-E, Plymouth',
    'mantel-box'                =>  '',
    'mantel-box2'               =>  '',
    'mantel-permutations'       =>  'Number of permutations.',
    'mantel-method_select'      =>  'Select correlation method, either "pearson", "spearman" or "kendall".',
    'metamds-function-title'    =>  'Nonmetric Multidimensional Scaling',
    'metamds-about'             =>  'This funtion performs Nonmetric Multidimensional Scaling (NMDS) and tries to find a stable solution using several random starts. In addition, it standardizes the scaling in the result, so that the configurations are easier to interpret, and adds species scores to the site ordination. Non-metric Multidimensional Scaling (NMDS) is commonly regarded as the most robust unconstrained ordination method in community ecology (Minchin 1987). 
    References: 
    Minchin, P.R. (1987) An evaluation of relative robustness of techniques for ecological ordinations. Vegetatio 69, 89–107.',
    'metamds-box'               =>  '',
    'metamds-box2'              =>  '',
    'metamds-transf_method_select'  =>  'Select a transformation method for the community data matrix. For an explanation of all the possibilities, please visit the taxondive fuction.',
    'metamds-transpose'         =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'metamds-column_select'     =>  'Select which column of the factor file you provided will be used as the colour coding factor of the plot.',
    'metamds-method_select'     =>  'Select the distance metric that will measure the dissimilarity between each pair of observations. For an explanation of all the possibilities, please visit the vegdist fuction.',
    'metamds-k_select'          =>  'Number of dimensions.',
    'metamds-trymax'            =>  'Maximum number of random starts in search of stable solution.',
    'metamds-autotransform_select'  =>  'If the data values are larger than common class scales, the function performs a Wisconsin double standardization using wisconsin. If the values look very large, the function also performs square root transformation. Both of these standardizations are generally found to improve the results. However, the limits are completely arbitrary (at present, data maximum 50 triggers square root and >9 triggers wisconsin). If you want to have a full control of the analysis, you should set autotransform = FALSE and make explicit standardization in the command.',
    'metamds-noshare'           =>  'Proportion of site pairs with no shared species to trigger stepacross function to find flexible shortest paths among dissimilarities. Ordination may be very difficult if a large proportion of sites have no shared species. In this case, the results may be improved with stepacross dissimilarities, or flexible shortest paths among all sites. The stepacross is triggered by option noshare. If you do not like manipulation of original distances, you should set noshare = 1.',
    'metamds-wascores_select'   =>  'Calculate species scores using function wascores.',
    'metamds-expand'            =>  'Expand weighted averages of species in wascores.',
    'metamds-trace'             =>  'Trace the function; trace = 2 or higher will be more voluminous.',
    'second_metamds-function-title'       =>  '2nd stage MDS',
    'second_metamds-about'                =>  'This function is computing inter-matrix correlations (‘similarities of similarity matrices’), using a correlation coefficient, to illustrate interactions between community data matrices (Clarke et al, 2006). The resulting triangular similarity matrix is displayed as an nMDS ordination plot.
	References:
	Clarke, K.R., Somerfield, P.J., Airoldi, L. & Warwick, R.M. (2006) Exploring interactions by second-stage community analyses. Journal of Experimental Marine Biology and Ecology 338(2), 179-192.',
    'second_metamds-box'        =>  '',
    'second_metamds-transf_method_select'   =>  'Select a transformation method for the community data matrices. For an explanation of all the possibilities, please visit the taxondive fuction.',
    'second_metamds-transpose'  =>  'Select if the matrices you uploaded had taxa as rows and samples as columns.',
    'second_metamds-method_select'  =>  'Select the distance metric that will measure the dissimilarity between each pair of observations for each community data matrix. For an explanation of all the possibilities, please visit the vegdist fuction.',
    'second_metamds-cor_method_select'  =>  '',
    'second_metamds-k_select'   =>  'Number of dimensions.',
    'second_metamds-trymax'     =>  'Maximum number of random starts in search of stable solution.',
    'second_metamds-autotransform_select'   =>  'If the data values are larger than common class scales, the function performs a Wisconsin double standardization using wisconsin. If the values look very large, the function also performs square root transformation. Both of these standardizations are generally found to improve the results. However, the limits are completely arbitrary (at present, data maximum 50 triggers square root and >9 triggers wisconsin). If you want to have a full control of the analysis, you should set autotransform = FALSE and make explicit standardization in the command.',
    'second_metamds-noshare'    =>  'Proportion of site pairs with no shared species to trigger stepacross function to find flexible shortest paths among dissimilarities. Ordination may be very difficult if a large proportion of sites have no shared species. In this case, the results may be improved with stepacross dissimilarities, or flexible shortest paths among all sites. The stepacross is triggered by option noshare. If you do not like manipulation of original distances, you should set noshare = 1.',
    'second_metamds-wascores_select'    =>  'Calculate species scores using function wascores.',
    'second_metamds-expand'     =>  'Expand weighted averages of species in wascores.',
    'second_metamds-trace'      =>  'Trace the function; trace = 2 or higher will be more voluminous.',
    'metamds_visual-function-title'       =>  'Alternative interactive visualization of the Nonmetric Multidimensional Scaling',
    'metamds_visual-about'      =>  'This function (SUMMARIZEplot) performs a Nonmetric Multidimensional Scaling of the most abundant taxa at the provided abundance matrix and returns interactive bar charts, pie chart and nMDS plot.',
    'metamds_visual-box'        =>  '',
    'metamds_visual-transf_method_select'   =>  'Select a transformation method for the community data matrix. For an explanation of all the possibilities, please visit the taxondive fuction.',
    'metamds_visual-transpose'  =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'metamds_visual-top_species'=>  'Define the number of most abundant taxa that should be used for the analysis.',
    'metamds_visual-method_select_viz'  =>  'Select the distance metric that will measure the dissimilarity between each pair of observations. For an explanation of all the possibilities, please visit the vegdist fuction.',
    'metamds_visual-k_select_viz'   =>  'Number of dimensions.',
    'metamds_visual-trymax_viz'     =>  'Maximum number of random starts in search of stable solution.',
    'cca_visual-function-title'       =>  '',
    'cca_visual-transf_method_select'   =>  '',    
    'cca_visual-about'      =>  '',
    'cca_visual-box'        =>  '',
    'cca_visual-box2'       =>  '',
    'cca_visual-transpose'      =>  '',
    'cca_visual-Factor_select1' =>  '',
    'cca_visual-Factor_select2' =>  '',
    'cca_visual-Factor_select3' =>  '',
    'radfit-function-title'       =>  'Rank – Abundance or Dominance / Diversity Models',
    'radfit-about'                =>  'This function constructs rank – abundance or dominance / diversity or Whittaker plots and fit brokenstick, pre-emption, log-Normal, Zipf and Zipf-Mandelbrot models of species abundance. Rank – Abundance Dominance (RAD) or Dominance/Diversity plots (Whittaker 1965) display logarithmic species abundances against species rank order. These plots are supposed to be effective in analysing types of abundance distributions in communities. These functions fit some of the most popular models mainly following Wilson (1991).
	References:
	Whittaker, R.H. (1965) Dominance and diversity in plant communities. Science 147, 250–260.
	Wilson, J.B. (1991) Methods for fitting dominance/diversity curves. Journal of Vegetation Science 2, 35–46.',
    'radfit-box'                =>  '',
    'radfit-transf_method_select'   =>  'Select a transformation method for the community data matrix. For an explanation of all the possibilities, please visit the taxondive fuction.',
    'radfit-transpose'          =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'radfit-column_radfit'      =>  'Select which row of the community matrix you uploaded (or which column of the transposed community matrix) should be used to construct the plots.',
    'bioenv-function-title'     =>  'BIO-ENV',
    'bioenv-about'              =>  'This function finds the best subset of environmental variables, so that the Euclidean distances of scaled environmental variables have the maximum (rank) correlation with community dissimilarities. Clarke & Ainsworth (1993) wrote a computer program BIO-ENV, thus giving the name to the current function. Presumably BIO-ENV was later incorporated in PRIMER software (available for Windows). In addition, Clarke & Ainsworth (1993) suggested a novel method of rank correlation which is not available in the current function.
	References:
	Clarke, K.R & Ainsworth, M. (1993) A method of linking multivariate community structure to environmental variables. Marine Ecology Progress Series 92, 205–219.',
    'bioenv-box'                =>  '',
    'bioenv-box2'               =>  '',
    'bioenv-transf_method_select'   =>  'Select a transformation method for the community data matrix. For an explanation of all the possibilities, please visit the taxondive fuction.',
    'bioenv-transpose'          =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'bioenv-method_select'      =>  'Select correlation method, either "pearson", "spearman" or "kendall".',
    'bioenv-index'              =>  'Select the distance metric that will measure the dissimilarity between each pair of observations in the community data matrix. For an explanation of all the possibilities, please visit the vegdist fuction.',
    'bioenv-upto'               =>  'Maximum number of environmental variables in studied subsets.',
    'bioenv-trace'              =>  'Trace the advance of calculations.',
    'simper-function-title'     =>  'Similarity Percentages',
    'simper-about'              =>  'This function attempts to assess the average percent contribution of individual variables to the dissimilarity between sampling units in a Bray-Curtis dissimilarity matrix. This allows users to identify variables that are likely to be the major contributors to any difference between groups detected by methods such as ANOSIM.',
    'simper-box'                =>  '',
    'simper-box2'               =>  '',    
    'simper-transpose'          =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'simper-column_select'      =>  'Select which column of the factor file you provided will be used as the grouping variable of the sampling units.',
    'simper-permutations'       =>  'The number of permutations required.',
    'simper-trace'              =>  'Trace permutations.',
    'regression-function-title' =>  '',
    'regression-about'          =>  '',
    'regression-box'            =>  '',
    'regression-transf_method_select'   =>  '',
    'regression-transpose'      =>  '',
    'regression-Factor_select1' =>  '',
    'regression-Factor_select2' =>  '',
    'regression-Factor_select3' =>  '',
    'parallel_taxa2dist-function-title'       =>  'Parallelized function for calculation of taxonomic distances',
    'parallel_taxa2dist-about'                =>  'This function is the parallelized implementation of the taxa2dist function. It should be used for medium (i.e. 16,900 rows) or large datasets (i.e. 42,300 rows). For smaller datasets (i.e. up to 1,700 rows), the serial version of the function should be used.',
    'parallel_taxa2dist-box'    =>  '',
    'parallel_taxa2dist-varstep'=>  'If TRUE, varies step lengths between successive levels relative to proportional loss of the number of distinct classes.',
    'parallel_taxa2dist-check_parallel_taxa2dist'   =>  'If TRUE, removes all redundant levels which are different for all rows or constant for all rows and regard each row as a different basal taxon (species). If FALSE all levels are retained and basal taxa (species) also must be coded as variables (columns).',
    'parallel_taxa2dist-No_of_processors'   =>  'Number of processors that should be used.',
    'parallel_postgres_taxa2dist-function-title'       =>  'PostgreSQL function for calculation of taxonomic distances',
    'parallel_postgres_taxa2dist-about'                =>  'This function is the PostgreSQL implementation of the taxa2dist function. It should be used for datasets larger than 42,300 rows, to avoid memory overflow of the parallel taxa2dist function.',
    'parallel_postgres_taxa2dist-box'       =>  '',
    'parallel_postgres_taxa2dist-varstep'   =>  'If TRUE, varies step lengths between successive levels relative to proportional loss of the number of distinct classes.',
    'parallel_postgres_taxa2dist-check_parallel_taxa2dist'  =>  'If TRUE, removes all redundant levels which are different for all rows or constant for all rows and regard each row as a different basal taxon (species). If FALSE all levels are retained and basal taxa (species) also must be coded as variables (columns).',
    'parallel_postgres_taxa2dist-No_of_processors'  =>  'Number of processors that should be used.',
    'parallel_anosim-function-title'       =>  'Parallelized function for Analysis of Similarities',
    'parallel_anosim-about'                =>  'This function is the parallelized implementation of the anosim function.',
    'parallel_anosim-box'       =>  '',
    'parallel_anosim-box2'      =>  '',
    'parallel_anosim-transpose' =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'parallel_anosim-No_of_processors'  =>  'Number of processors that should be used.',
    'parallel_anosim-column_select'     =>  'Select which column of the factor file you provided will be used as the grouping variable of the sampling units.',
    'parallel_anosim-permutations'      =>  'Number of permutation to assess the significance of the ANOSIM statistic.',
    'parallel_anosim-method_select'     =>  'Select the distance metric that will measure the dissimilarity between each pair of observations. For an explanation of all the possibilities, please visit the vegdist fuction.',
    'parallel_mantel-function-title'       =>  'Parallelized function for Mantel Tests for Dissimilarity Matrices',
    'parallel_mantel-about'                =>  'This function is the parallelized implementation of the mantel function.',
    'parallel_mantel-box'               =>  '',
    'parallel_mantel-box2'              =>  '',
    'parallel_mantel-No_of_processors'  =>  'Number of processors that should be used.',
    'parallel_mantel-permutations'      =>  'Number of permutations.',
    'parallel_mantel-method_select'     =>  'Select correlation method, either "pearson", "spearman" or "kendall".',
    'parallel_taxa2taxon-function-title'       =>  'Parallelized function for calculation of taxonomic distances and Indices of Taxonomic Diversity and Distinctness',
    'parallel_taxa2taxon-about'                =>  'This function is the parallelized implementation of the taxa2dist and taxondive functions combined.',
    'parallel_taxa2taxon-box'           =>  '',
    'parallel_taxa2taxon-box2'          =>  '',
    'parallel_taxa2taxon-transpose'     =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'parallel_taxa2taxon-No_of_processors'  =>  'Number of processors that should be used.',
    'parallel_taxa2taxon-varstep'       =>  'If TRUE, varies step lengths between successive levels relative to proportional loss of the number of distinct classes.',
    'parallel_permanova-function-title'       =>  'Parallelized function for Permutational Multivariate Analysis of Variance Using Distance Matrices',
    'parallel_permanova-about'                =>  'This function is the parallelized implementation of the adonis function.',
    'parallel_permanova-box'            =>  '',
    'parallel_permanova-box2'           =>  '',
    'parallel_permanova-transpose'      =>  'Select if the matrix you uploaded had taxa as rows and samples as columns.',
    'parallel_permanova-column_select'  =>  'Select which column of the factor file you provided will be used as the grouping variable of the sampling units.',
    'parallel_permanova-column_select2' =>  'Select which column of the factor file you provided will be used as the second grouping variable of the sampling units, in case you choose the multiple parameter option.',
    'parallel_permanova-permutations'   =>  'Number of replicate permutations used for the hypothesis tests (F tests).',
    'parallel_permanova-method_select'  =>  'Select the method used to calculate pairwise distances between each pair of observations. For an explanation of all the possibilities, please visit the vegdist fuction.',
    'parallel_permanova-No_of_processors'   =>  'Number of processors that should be used.',
    
    'mapping_tools_visual-function-title'       =>  '',
    'mapping_tools_visual-about'                =>  '',
    'mapping_tools_visual-box'                  =>  '',
    'mapping_tools_visual-box2'                 =>  '',
    'mapping_tools_visual-transf_method_select' =>  '',
    'mapping_tools_visual-transpose'            =>  '',
    'mapping_tools_visual-top_species'          =>  '',
    
    'mapping_tools_div_visual-function-title'       =>  '',
    'mapping_tools_div_visual-about'                =>  '',
    'mapping_tools_div_visual-box'                  =>  '',
    'mapping_tools_div_visual-box2'                 =>  '',
    'mapping_tools_div_visual-box3'                 =>  '',
    'mapping_tools_div_visual-transf_method_select' =>  '',
    'mapping_tools_div_visual-transpose'            =>  '',
    'mapping_tools_div_visual-column_select'        =>  '',
    'mapping_tools_div_visual-top_species'          =>  '',
    
    'parallel_simper-function-title'        =>  '',
    'parallel_simper-about'                 =>  '',
    'parallel_simper-box'                   =>  '',
    'parallel_simper-box2'                  =>  '',
    'parallel_simper-transpose'             =>  '',
    'parallel_simper-No_of_processors'      =>  '',
    'parallel_simper-column_select'         =>  '',
    'parallel_simper-permutations'          =>  '',
    'parallel_simper-trace'                 =>  '',
    
    'parallel_bioenv-function-title'       =>  '',
    'parallel_bioenv-about'                =>  '',
    'parallel_bioenv-box'                  =>  '',
    'parallel_bioenv-box2'                 =>  '',
    'parallel_bioenv-transpose'            =>  '',
    'parallel_bioenv-No_of_processors'     =>  '',
    'parallel_bioenv-method_select'        =>  '',
    'parallel_bioenv-index_select'         =>  '',
    'parallel_bioenv-upto'                 =>  '',
    'parallel_bioenv-trace'                =>  '',
    
    'convert_2_r-header1_id'                =>  '',
    'convert_2_r-header2_id'                =>  '',
    'convert_2_r-header3_id'                =>  '',
    'convert_2_r-function_to_run'           =>  '',
    'convert_2_r-header1_fact'              =>  '',
    'convert_2_r-header2_fact'              =>  '',
    'convert_2_r-header3_fact'              =>  ''
);